grapple/actions.py

```diff
@@ -268,7 +269,13 @@ class StubMeta:
         model = stub_model
 
     # Gather any interfaces, and discard None values
-    interfaces = {interface, *getattr(cls, "graphql_interfaces", ())}
+    interface_classes = list(getattr(cls, "graphql_interfaces", ()))
+    for i, interface_class in enumerate(interface_classes):
+        if isinstance(interface_class, str):
+            interface_classes[i] = import_string(interface_class)
+    interface_classes = tuple(interface_classes)
+
+    interfaces = {interface, *interface_classes}
```

Comment:
There's some unnecessary construction of intermediate containers here:

1. cast `interface_classes` to a list;
2. mutate it;
3. cast it to a tuple;
4. use the tuple elements in a new set.

The desired behaviour might be expressed more simply, something like:

```suggestion
    interface_classes = getattr(cls, "graphql_interfaces", ())
    interfaces = {
        import_string(i) if isinstance(i, str) else i for i in interface_classes
    }
    interfaces.add(interface)
```